src/BuilderMacros.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Laravel Scout Extended.
+ *
+ * (c) Algolia Team <contact@algolia.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace Algolia\LaravelScoutExtended;
+
+use Closure;
+use function get_class;
```

Comment:
Is this required or is it added to make highlight the method used in this class?

Reply:
Is not required. Is basically to leverage some opcode optimisations. It's also to highlight that we use that method on the class.